### PR TITLE
Enable Facility Cover Image Uploads on Mobile Screens

### PR DESCRIPTION
**Pull request by gigincg**
## Proposed Changes

- Fixes cover image edit for mobile devices
- Remove imageKey Implementation for Cover Images
- Fix `Edit Cover Image` for Mobile Devices

@rithviknishad Can you confirm that removing the imageKey wouldn't cause any issues

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — DISMISSED** [review]

**Comment by rithviknishad** [comment]
> @rithviknishad Can you confirm that removing the imageKey wouldn't cause any issues

@gigincg `?imageKey` was originally intended to force refresh the image as the signed URL for a facility wouldn't change even when the cover image was changed.

**Comment by nihal467** [comment]
LGTM

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -95,8 +97,8 @@ export const FacilityHome = (props: any) => {
   const hasCoverImage = !!facilityData?.read_cover_image_url;
 
   const StaffUserTypeIndex = USER_TYPES.findIndex((type) => type === "Staff");
-  const hasPermissionToEditCoverImage =
-    !(authUser.user_type as string).includes("ReadOnly") &&
+  const hasPermissionToEditCoverImage = true;
+  !(authUser.user_type as string).includes("ReadOnly") &&
```
Comment (khavinshankar) [inline src/Components/Facility/FacilityHome.tsx]: Was this `true` experimental?

Comment (gigincg) [reply src/Components/Facility/FacilityHome.tsx]: @khavinshankar Yes, lemme fix it

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Pull request merged**